src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -313,6 +313,12 @@ private object Template {
        |}
        |""".stripMargin
 
+  def shellPromptWithGit: String =
+    """|import com.typesafe.sbt.SbtGit.GitCommand
```

Comment:
Sounds good to me. :-)

I would suggest adding the prompt to the common settings or enable the plugin if desired.

Shall I rework the pull-request to model these changes?